tests/sentry/api/endpoints/test_organization_plugins_configs.py

```diff
@@ -61,13 +61,13 @@ def test_only_configuable_plugins(self):
     def test_enabled_not_configured(self):
         plugins.get("webhooks").enable(self.projectA)
         response = self.client.get(self.url)
-        assert filter(lambda x: x["slug"] == "webhooks", response.data)[0]["projectList"] == []
+        assert next(filter(lambda x: x["slug"] == "webhooks", response.data))["projectList"] == []
```

Comment:
`next` fails here in py2, since filter produces a list rather than an iterator.

Reply:
damn why did I even try to do this lol

Reply:
Because I was being a smartass on one of your diffs and now infected your brain @EvanPurkhiser 🤣